tests/PHPCR/Tests/Util/Console/Command/NodesUpdateCommandTest.php

```diff
@@ -88,7 +88,7 @@ public function testNodeUpdate($options)
         $this->setupQueryManager($options);
 
         $args = array(
-            '--query-language' => null,
+            '--query-language' => 'jcr-sql2',
```

Comment:
this is the default value in the command - when the null is passed the command throws an error.